docs/index.rst

```diff
@@ -1,17 +1,67 @@
 .. include:: ../README.rst
    :end-before: About
 
+
 .. include:: ../README.rst
-   :start-after: -----
+   :start-line: 25
    :end-before: Features
 
-.. toctree::
-   :numbered:
 
-   introduction
-   cli
-   restapi
-   contributing
-   changes
-   license
-   authors
+.. include:: ../README.rst
+   :start-line: 32
+   :end-before: Useful links
+
+**Contents**
+
+.. contents::
+   :local:
+   :depth: 1
+   :backlinks: none
+
+
+CLI API
+=======
+
+.. click:: reana_server.reana_admin:reana_admin
+   :prog: flask reana-admin
+   :show-nested:
+
+
+REST API
+========
+
+The REANA Server offers a REST API for management workloads
+(workflows, jobs, tasks, etc.) running on REANA Cloud.
+Detailed REST API documentation can be found `here <_static/api.html>`_.
+
+.. automodule:: reana_server.rest.ping
+   :members:
+
+.. automodule:: reana_server.rest.users
+   :members:
+
+.. automodule:: reana_server.rest.workflows
+   :members:
+
+
+.. include:: ../CONTRIBUTING.rst
+
+.. include:: ../CHANGES.rst
```

Comment:
Please put CHANGES before CONTRIBUTING.  The thinking goes to group all admin-usage sections (such as CLI API, REST API, change list) before developer-usage sections (issues, pull requests, license, authors).